### PR TITLE
Update the version to 1.0.0 from 0.0.1

### PR DESCRIPTION
**Pull request by hdost**
* Don't check the version for now to easy allow for OM rollout

Relates to https://github.com/prometheus/prometheus/issues/9430

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1:

File: expfmt/encode.go [hunk 1]
```diff
@@ -99,7 +99,8 @@ func NegotiateIncludingOpenMetrics(h http.Header) Format {
 		if ac.Type == "text" && ac.SubType == "plain" && (ver == TextVersion || ver == "") {
 			return FmtText
 		}
-		if ac.Type+"/"+ac.SubType == OpenMetricsType && (ver == OpenMetricsVersion || ver == "") {
+		// Intentionally Not Checking Version to Allow for OpenMetics rollout
```
Comment (roidelapluie) [inline expfmt/encode.go:102]:
```suggestion
		// Intentionally not checking version to allow for OpenMetics rollout.
```

**Comment by mrueg** [comment]
@hdost any chance to apply @roidelapluie's suggestion here? It would be good to have it updated.

**Comment by hdost** [comment]
> @hdost any chance to apply @roidelapluie's suggestion here? It would be good to have it updated.

Indeed, sorry for the delay

**Comment by hdost** [comment]
@mrueg done btw, squashed it today.

**Comment by SuperQ** [comment]
This needs a rebase.

**Comment by hdost** [comment]
Seems like maybe https://github.com/prometheus/common/pull/475 superseded this? 😓

**Comment by SuperQ** [comment]
Oops, sorry about that.

**Pull request closed without merging**